README.md

```diff
@@ -96,6 +96,8 @@ library for it project:
   enabled on buffers.
 * **WITH_DCACHE_RSC_TABLE** (default OFF): Build with data cache operations
   enabled on resource table.
+* **WITH_DCACHE** (default OFF): Build with all cache operations
+  enabled.
```

Comment:
Please indicate explicitly the list of option that are forced to enable, something like that...

When set to ON,  **WITH_DCACHE_VRINGS**, **WITH_DCACHE_RSC_TABLE** and **WITH_DCACHE** are forced to ON.

Reply:
done